pages/portfolio.html

```diff
@@ -5,8 +5,70 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <nav class="nav">
+    <ul>
+      <li><a href="../index.html">Home</a> </li>
+      <li><a href="about.html">About</a> </li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+    </ul>
+      </nav>
+    <div class="wrapper">
+      <section>
+        <h2> Projects completed by Roza Ajema</h2>
+        <p class="paragraph">
+        Below are the projects that Roza completed while at Ada. Github references are also presented here. Some of the projects
+        completed by pair programming, usually with one other adie.  
+        </p>
+        <ul class = "port">
```

Comment:
This probably should have been a few separate `<section>`s (one for each project) with a `<h3>` with each project's name and `<p>`s describing them.